.github/workflows/test.yml

```diff
@@ -514,6 +520,30 @@ jobs:
           echo OVN_TEST_EX_GW_NETWORK=kindexgw >> $GITHUB_ENV
           echo OVN_ENABLE_EX_GW_NETWORK_BRIDGE=true >> $GITHUB_ENV
         fi
+        if [[ "$JOB_NAME" == *"shard-conformance"* ]] && [ "$ADVERTISE_DEFAULT_NETWORK" == "true" ]; then
```

Comment:
We cannot have JOB_NAME at kind.sh

Reply:
yup yup we can just need to check if its set or not :) https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5121#discussion_r2000564580

Reply:
this is done